packages/material-ui/src/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -45,4 +47,52 @@ describe('<Breadcrumbs />', () => {
     assert.strictEqual(wrapper.is('.test-class-name'), true);
     assert.strictEqual(wrapper.hasClass(classes.root), true);
   });
+
+  it('should render seperators', () => {
+    const wrapper = shallow(
+      <Breadcrumbs>
+        <span />
+        <span />
+        <span />
+      </Breadcrumbs>,
+    );
+    assert.strictEqual(wrapper.find(BreadcrumbSeparator).length, 2);
+  });
+
+  it('should render an ellipse', () => {
+    const wrapper = shallow(
+      <Breadcrumbs>
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+      </Breadcrumbs>,
+    );
+    assert.strictEqual(wrapper.find(BreadcrumbSeparator).length, 2);
+    assert.strictEqual(wrapper.find(BreadcrumbCollapsed).length, 1);
+  });
+
+  it('should expand', () => {
+    const wrapper = shallow(
+      <Breadcrumbs>
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+        <span />
+      </Breadcrumbs>,
+    );
+    assert.strictEqual(wrapper.state().expanded, false);
+    wrapper.find(BreadcrumbCollapsed).simulate('click');
+    assert.strictEqual(wrapper.state().expanded, true);
```

Comment:
I think we should actually test that it does not render every component and will render them after the click. Testing internal state is very brittle.